addon/services/page-title-list.js

```diff
@@ -251,4 +249,18 @@ export default class PageTitleListService extends Service {
       document.head.querySelectorAll('title').length === 1
     );
   }
+
+   /**
+   * Find token by id
+   *
+   * IE11 compatible approach due to lack of Array.find support
+   *
+   * @param {String} id
+   * @private
+   */
+  _findTokenById(id) {
+    return this.tokens.filter((token) => {
+      return token.id === id;
+    })[0];
```

Comment:
Should this check length before accessing the first index?

Reply:
Empty array is always returned - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

Reply:
This will end up as `let arr = []; arr[0] => undefined`

Reply:
Ah, yes, forgot that was how index access worked. 👍